dags/analyzer_helper/tests/unit/test_load_transformed_data_base.py

```diff
@@ -23,18 +23,22 @@ def test_get_platform_id(self):
         """
         self.assertEqual(self.test_instance.get_platform_id(), self.platform_id)
 
-    @patch('from dags.analyzer_helper.discord.load_transformed_data_base.LoadTransformedDataBase')
+    @patch.object(LoadTransformedDataBase, 'load_impl')
     def test_load(self, mock_load_impl):
         """
         Tests that the load method calls the load_impl method with the correct arguments
         """
         self.test_instance.load(self.processed_data)
         mock_load_impl.assert_called_once_with(self.processed_data, self.platform_id, False)
 
-    @patch('from dags.analyzer_helper.discord.load_transformed_data_base.LoadTransformedDataBase')
+    @patch.object(LoadTransformedDataBase, 'load_impl')
     def test_load_with_recompute(self, mock_load_impl):
         """
         Tests that the load method calls the load_impl method with recompute=True when specified
         """
         self.test_instance.load(self.processed_data, recompute=True)
         mock_load_impl.assert_called_once_with(self.processed_data, self.platform_id, True)
+
+
+if __name__ == '__main__':
```

Comment:
Please consider removing this part

You can always run your test cases via the command below

```
pytest [test_case.py]
```

In this repository you can run the test cases locally from the directory `dags`